docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst

```diff
@@ -0,0 +1,63 @@
+|Google logo|
+
+===============================
+Creating a File Accessor Plugin
+===============================
+
+.. container::
+
+   .. container:: content
+
+      .. rubric:: Experimental
+
+      Please note that this functionality is currently experimental. Creating
+      and using a plugin is not guaranteed to work as expected at this time.
+
+      .. rubric:: Description
+
+      GEE Server implements a plugin system for file accessors. That is, plugins
```

Comment:
Minor: Do you think that we should make it clearer that not all of GEE Server has been updated to work with the plugin yet? As in, some parts of the code use it and some don't? I could go either way.

Reply:
That's probably a good idea. I'll update it. We just have to remember to update it as their use expands.